__tests__/app.test.js

```diff
@@ -12,6 +12,24 @@ beforeEach(() => {
     return seed(testData)
 })
 
+describe("/api", () => {
+    test("Get: 200 sends an object of all available endpoints", () => {
+        return request(app)
```

Comment:
You could make this 200 test for `GET /api` more dynamic (accurate to all updates) by requiring in the `endpoints.json` file directly to get access to the whole object, then expecting that the response.body is equal to it.